src/scripts.js

```diff
@@ -24,11 +24,35 @@ const loadPageHandler = () => {
   findTags(recipeData)
 }
 
+const reformatData = () => {
```

Comment:
This is dope. also noticed the awesome thing your doing that makes this happen on load pretty much.